src/Nethermind/Nethermind.Serialization.Rlp/IByteBufferExtensions.cs

```diff
@@ -41,5 +42,18 @@ public static void MakeSpace(this IByteBuffer output, int length, string reason
                 }
             }
         }
+
+        /// <summary>
+        /// Return readable space of this byte buffer as a span.
+        /// </summary>
+        /// <param name="buffer"></param>
+        /// <returns></returns>
+        /// <exception cref="InvalidOperationException"></exception>
```

Comment:
do we not need to pass `_initialPosition` in some `start` param?

Reply:
Yea, I'm on the fence with this. If its just `IByteBuffer`, then no start param. I guess with `NettyRlpStream`, the logic is different.

Reply:
i'll just add an optional param.